src/services/emptyDocuments.ts

```diff
@@ -96,7 +96,6 @@ const emptyDocuments: Document[] = [
     eventStatus: "CLOSED",
```

Comment:
Not part of this PR, however it might be nice to update the default date here `date: 1442430000000` to be one that is this year (something like 1722470400000), so it's better for creating new dates. It doesn't really make sense to have this default to 2015 now.